### PR TITLE
Merge development to master

### PR DESCRIPTION
**Pull request by Abdusaid10**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2102 @@
...
+                                        White House Press Secretary Stephanie Grisham Reportedly
+                                        Involved in Scuffle With North Korean Security at DMZ
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+
+                    </article>
+                </div>
+                <div class="more-stories">
```
Comment (Torres-ssf) [inline index.html]: Use section tag instead.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2102 @@
...
+                </div>
+            </nav>
+        </div>
+    </header>
+
+    <main class="container-fluid bg-white">
+        <div class="row">
+           
+            <div class="col-md-5 col-lg-4 order-md-5-1 order-2 ">
+                <div class="featured">
```
Comment (Torres-ssf) [inline index.html]: Use section tag instead.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,2102 @@
...
+                            <div class="card-text px-0 py-2">Working with Statista Inc., the globally recognized data
+                                research
+                                company, Newsweek has produced an authoritative list of America's Best Business Tools.
+                            </div>
+                        </div>
+
+                    </article>
+                </div>
+
+                <div class="sponsor p-3 ">
```
Comment (Torres-ssf) [inline index.html]: Use section tag instead.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,2102 @@
...
+                                are producing graduates that will transform our world?
+                            </p>
+                        </div>
+
+                    </article>
+                </div>
+
+            </div>
+            <div class="col-md-7 col-lg-5 order-md-7-2 order-1">
+                <div class="top-story">
```
Comment (Torres-ssf) [inline index.html:278]: Use section tag instead.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,2102 @@
...
+                                22 European Countries Still Won't
+                                Say Sex Without Consent is Rape
+                            </a>
+                            <div class="text-uppercase font-weight-bolder text-black-50 pt-1 pb-2">
+                                By Stefan Simanowitz
+                            </div>
+                        </div>
+                    </li>
+                </ul>
+                <div class="latest-news">
```
Comment (Torres-ssf) [inline index.html]: Use section tag instead.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,2102 @@
...
+                            </div>
+                            <h6 class="media-info">
+                                <a class="font-weight-bolder" href="#">
+                                    Trump Team Is Tackling Migration the 'Wrong Way'—El Salvador President
+                                </a>
+                            </h6>
+                        </li>
+                    </ul>
+                </div>
+                <div class="sponsored-right py-3 mx-auto">
```
Comment (Torres-ssf) [inline index.html]: Use section tag instead.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,2102 @@
...
+                                Business schools offer more than just certification—they nurture the leaders of
+                                tomorrow.
+                                But which schools, in which countries,
+                                are producing graduates that will transform our world?
+                            </p>
+                        </div>
+                    </article>
+                </div>
+              
+                <div class="card mx-auto my" style="width: 16rem;">
```
Comment (Torres-ssf) [inline index.html]: Use section tag instead.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,2102 @@
...
+                        </div>
+                        <button class="btn btn-secondary text-uppercase font-weight-bold px-4 my-2 rounded-0">
+                            sign up >
+                        </button>
+                        <p class="font-weight-bold">Update your preference >></p>
+
+                    </div>
+                </div>
+              
+                <div class="special-edition py-3 mx-auto">
```
Comment (Torres-ssf) [inline index.html]: Use section tag instead.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,2102 @@
...
+                                tomorrow.
+                                But which schools, in which countries,
+                                are producing graduates that will transform our world?
+                            </p>
+                        </div>
+                    </article>
+                </div>
+            </div>
+        </div>
+        <div class="page-bottom pt-3 mb-3">
```
Comment (Torres-ssf) [inline index.html]: You need to create 3 sections here. One for 'In the Magazine', another one for 'Editor's Pick' and finally one for 'Featured Slideshow'. Remember that each article must be inside an article tag. The article includes one whole content, like the heading, the image and the description of the news.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]

**Comment by Torres-ssf** [comment]
Hi there @Abdusaid10 
You are doing a great on this page and should be proud 👍. But before submitting to master we need to do some changes to make it match Microverse Guidelines.

- [ ] All sections (Top Story, More Stories, Featured Stories, Opinion, Latest News, In the Magazine, Editor's picks, Featured Slideshow) must be section tag. And all articles inside it must be inside an article tag.

- [ ] Your opinion section also must be inside a section tag. Kindly put the ul tag inside a section tag. You also need to put each one of the opinion articles inside an article tag. 

- [ ] There is a problem with you navbar at small resolutions, specific with the 'sign in' and 'subscribe button'. Your navbar needs to get the full available width in all resolutions. Try to remove margins and paddings to fix that.

- [ ] I also left some reviews in your code.

Plese, kindly make these changes and submit for code review again.

**Comment by maelfosso** [comment]
Hello,

#  GOOD JOB 

You can fill a project completion form.

**Pull request merged**